drivers/gpio/gpio_sam0.c

```diff
@@ -113,23 +183,78 @@ static int gpio_sam0_read(struct device *dev, int access_op, u32_t pin,
 	return 0;
 }
 
+#ifdef CONFIG_SAM0_EIC
+
+int gpio_sam0_manage_callback(struct device *dev,
+			      struct gpio_callback *callback, bool set)
+{
+	struct gpio_sam0_data *const data = DEV_DATA(dev);
+
+	return _gpio_manage_callback(&data->callbacks, callback, set);
```

Comment:
This is `gpio_manage_callback` now.

Reply:
Good catch on both.  I've fixed it and rebased/retested on the current master.